doc/usecases/cli/UC_namespace.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: cli extract namespace
```

Comment:
I don't think this is an extra use case, it is rather a part of the use case to print key names.